Makefile

```diff
@@ -28,6 +28,7 @@ $(MGEN): ./msgp/defs_test.go
 
 test: all
 	go test -v ./msgp
+	go test -v ./gen
```

Comment:
You should add this line to the `travis` rule as well (sorry; the Makefile is a mess, as it always is...)